packages/resourceful-action-creators/test/unit/resourceful-action-creators.js

```diff
@@ -473,7 +473,7 @@ describe('Resourceful Action Creators', function() {
             url: 'http://www.google.com',
             method: 'GET',
           }
-        });
+        }, undefined);
```

Comment:
There's no need to explicitly pass `undefined` here, since the code already was doing this